trade-finance-manager-ui/component-tests/utilisation-reports/_macros/report-reconciliation-table.component-test.js

```diff
@@ -57,6 +58,8 @@ describe(component, () => {
       wrapper.expectElement(`${rowSelector} th`).toHaveCount(1);
       wrapper.expectElement(`${rowSelector} th:contains("${summaryItem.bank.name}")`).toExist();
 
+      wrapper.expectText(`${rowSelector} th > p`).toRead(summaryItem.bank.name);
```

Comment:
Have you checked if this fails if the bank name is a link, e.g. if you set the feature flag to true at the top of this test and run it does it fail?

Reply:
It does indeed pass even if the feature flag is set to true. I added an extra line to check that the `<a>` element does not exist and then it works as intended